build/configs/artik05x/artik05x_download.sh

```diff
@@ -67,10 +69,28 @@ Options:
 EOF
 }
 
+prepare_resource()
+{
+    if [ -d "${RESOURCE_DIR_PATH}" ]; then
+        echo "Packing resources into romfs.img ..."
+        # When CONGIG_FRAME_POINTER flag is enabled, copy the build/output/bin/System.map file
+        # to tools/fs/contents directory so that symbol names can be displayed using dump_stack
+        if [ "${CONFIG_FRAME_POINTER}" == "y" ]; then
+            echo "Copying System.map file"
```

Comment:
Shall we add ${RESOURCE_DIR_PATH} as well in echo message.
Like 
echo "Copying System.map file to ${RESOURCE_DIR_PATH}"